go.mod

```diff
@@ -9,11 +9,11 @@ require (
 	github.com/googleapis/gnostic v0.2.0 // indirect
 	github.com/gravitational/logrus v0.10.1-0.20180402202453-dcdb95d728db
 	github.com/gravitational/trace v0.0.0-20190218181455-5d6afe38af2b
-	github.com/jrivets/log4g v0.0.0-20171008071556-0d3a418ba12a
+	github.com/jrivets/log4g v0.0.0-20191016233753-c02c5046dc98
 	github.com/json-iterator/go v1.1.6 // indirect
 	github.com/julienschmidt/httprouter v1.2.0
-	github.com/logrange/logrange v0.1.43
-	github.com/logrange/range v0.0.0-20190621064142-ba348b8545e2
+	github.com/logrange/logrange v0.1.46
```

Comment:
I see the vendored version here changed, but there are no associated code changes?

Reply:
it's the latest fix that Dmitry introduced to workaround apimachinery breaking. That's why the update is there.

Reply:
Yes, I know, but where are the revendored code changes then?

Reply:
This needs a run of `go mod tidy` / `go mod vendor`